include/libvfio-user.h

```diff
@@ -88,6 +88,10 @@ dma_sg_size(void);
  */
 #define LIBVFIO_USER_FLAG_ATTACH_NB  (1 << 0)
 
+// Should the server start in the migration "RESUMING" state? Given to
```

Comment:
Is it clearer to say "If set in the flags given to `init_migration`, the server will start in the RESUMING state, ready to receive migration data."?

Reply:
Yes. Though I just realized, how does it work in VFIO? If transitioning from `RUNNING` to `RESUMING`, then how can the destination device ever get into the `RESUMING` state?

Reply:
Yes to make the transition you have to go through the whole sequence, so I thought it made more sense to just be able to start the destination in the RESUMING state as it doesn't make sense for it to go through the full sequence of saving states just to receive data?

Reply:
If that's how a destination device ends up in `RESUMING` then we should do the same.

Reply:
@jlevon What do you think we should do about this? In our sample, we require the client to consume all of the migration data stream before it can move to the `STOP` state, so doing this whole sequence of states would require us to consume and then discard all the migration data just to get the server into the `RESUMING` state - personally I think it makes much more sense to use the flag to start the destination in `RESUMING`.